modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/HelpText/HelpText.es.js

```diff
@@ -0,0 +1,259 @@
+					<kbd className="c-kbd c-kbd-light">[</kbd>
+					<span className="c-kbd-separator"> </span>
+					<kbd className="c-kbd c-kbd-light">]</kbd>
+					<span className="c-kbd-separator"> </span>
+					<kbd className="c-kbd c-kbd-light">&#x7B;</kbd>
+					<span className="c-kbd-separator"> </span>
+					<kbd className="c-kbd c-kbd-light">&#x7D;</kbd>
+					<span className="c-kbd-separator"> </span>
+					<span className="c-kbd-separator">
+						{' '}
+						{Liferay.Language.get('group-separators')}
+					</span>
+				</kbd>
+			</div>
+			<div className="lfr-ddm-help-text-line">
+				<kbd className="c-kbd">
+					<kbd className="c-kbd c-kbd-light">#</kbd>
+					<span className="c-kbd-separator"> </span>
+					<kbd className="c-kbd c-kbd-light">$</kbd>
+					<span className="c-kbd-separator"> </span>
+					<kbd className="c-kbd c-kbd-light">%</kbd>
+					<span className="c-kbd-separator"> </span>
+					<kbd className="c-kbd c-kbd-light">+</kbd>
+					<span className="c-kbd-separator"> </span>
+					<span className="c-kbd-separator">
+						{' '}
+						{Liferay.Language.get('prefix-and-suffix-symbols')}
+					</span>
+				</kbd>
+			</div>
+		</div>
+	);
+
+	return (
+		<div
+			aria-labelledby={!renderLabel ? fieldDetailsId : null}
+			className={classNames('form-group', {
+				'has-error': hasError,
+				hide: !visible,
+			})}
+			data-field-name={name}
+			style={style}
+			tabIndex={!renderLabel ? 0 : null}
+		>
+			<div className="lfr-ddm-help-text-flex-container">
+				<div className="lfr-ddm-help-text-flex-element">
+					{renderLabel && (
+						<>
+							<label
+								aria-describedby={fieldDetailsId}
+								className={classNames(
+									'lfr-ddm-help-text-label'
+								)}
+								tabIndex="0"
+							>
+								{label && showLabel && label}
+							</label>
+						</>
+					)}
+				</div>
+				<div className="lfr-ddm-help-text-flex-element">
+					<ClayButton
+						borderless={true}
+						className="lfr-ddm-help-text-button"
+						disabled={readOnly}
+						displayType="secondary"
+						name="enableHelpText"
+						onClick={() => {
+							setEnableHelpText(!enableHelpText),
+								setShowHideButton(
+									!enableHelpText ? 'Show less' : 'Show more'
+								);
+						}}
+						type="button"
+					>
+						{showHideButton}
+					</ClayButton>
+				</div>
+			</div>
+
+			{enableHelpText && helpText}
+		</div>
+	);
+};
+
+const Main = ({
+	dataType: initialDataType,
+	defaultLanguageId,
+	editingLanguageId,
+	label,
+	name,
+	onChange,
+	readOnly,
+	spritemap,
+	value = {},
+	visible,
+}) => {
+	const data = transformData({
+		defaultLanguageId,
+		editingLanguageId,
+		initialDataType,
+		value,
+	});
+
+	return (
+		<HelpText
+			{...data}
+			defaultLanguageId={defaultLanguageId}
+			editingLanguageId={editingLanguageId}
+			label={label}
+			name={name}
+			onChange={(value) => onChange({}, value)}
+			readOnly={readOnly}
+			spritemap={spritemap}
+			value={value}
+			visible={visible}
+		/>
```

Comment:
@mateusgomes01 Please remove the props that you don't need in the Help Text field.

Reply:
Ok!